README.md

```diff
@@ -11,12 +11,10 @@ The code is currently aligned with [Bootstrap v4.1.3](https://github.com/twbs/bo
 
 ## Getting Started
 
-Options for installing Bootstrap Less in your project:
+Options for installing Bootstrap Less in your project (Less v3.7+ required):
 
-- Install with [npm](https://www.npmjs.com/): `npm install bootstrap-less-port`
-- Install with [yarn](https://yarnpkg.com/): `yarn add bootstrap-less-port`
-- Clone the repo: `git clone https://github.com/seanCodes/bootstrap-less-port.git`
-- [Download the latest release](https://github.com/seanCodes/bootstrap-less-port/archive/master.zip)
+- Install with [npm](https://www.npmjs.com/): `npm install less bootstrap-less-port`
+- Install with [yarn](https://yarnpkg.com/): `yarn add less bootstrap-less-port`
```

Comment:
Makes sense to separate. Should probably be something like  `npm install less@latest` somewhere. (Or whatever appropriate NPM command to make sure their version conforms to 3.8.1+? Not sure what `npm install less@^3.8.1` does, for example, if you have Less 3.9 installed. But it would be nice if it pushed the version UP if they already have it in their dependencies. But you're right that a peer dependency warning will show non-conformity. NPM will say something like `bootstrap-less-port requires less ^3.8.1 and you have 3.0.0 installed`

Reply:
@matthew-dean Sounds good, I’ll add the install Less step after we merge this PR.

As for the #13 confusion, I asked @calvinjuarez to split the original PRs into two so that we can have a version of the port tied to Less 3.8 and one tied to Less 3.9

Reply:
Oh interesting. Okay.

Reply:
For the mention of installing Less, @seanCodes what do you recommend.

Reply:
@calvinjuarez I think `npm install less@^3.8.1` would do the trick. I’ll make the change on your branch.